apps/server/src/modules/files-storage/controller/api-test/files-storage-download-upload.api.spec.ts

```diff
@@ -376,35 +377,64 @@ describe('files-storage controller (API)', () => {
 		});
 
 		describe(`with valid request data`, () => {
-			const setup = async () => {
-				const { result: uploadedFile } = await api.postUploadFile(`/file/upload/${validId}/schools/${validId}`);
-				const expectedResponse = TestHelper.createFile('bytes 0-3/4');
+			describe('when mimetype is not application/pdf', () => {
```

Comment:
I think this describe should only be around the test for the header. Otherwise it looks like the tests for the statuses would only be for this scenario.

Reply:
No, they are not part of this particular scenario. The mimetype does not affect the return statuses.

Reply:
They cluster behavior. You don't create a scenario for every possible variation of inputs. (In most cases this would mean infinitely many scenarios.) You create scenarios where you expect distinct behavior.